modules/virt-creating-configmap.adoc

```diff
@@ -1,36 +1,136 @@
 ----
 +
-If the RHV VM operating system does not have a corresponding {VirtProductName} template, you must create a custom ConfigMap.
+The output is similar to the following example:
++
+----
+fedora31
+fedora32
+...
+rhel8.1
+rhel8.2
+...
+----
+
+. If a template corresponding to the RHV VM operating system does not appear in the list of available templates, create a template with the {VirtProductName} web console.
+
+. Create a ConfigMap to map the RHV VM operating system to the template:
++
+[source,yaml]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: os-configmap
+  namespace: default <1>
+data:
+  guestos2common: |
+    "Red Hat Enterprise Linux Server": "rhel"
+    "CentOS Linux": "centos"
+    "Fedora": "fedora"
+    "Ubuntu": "ubuntu"
+    "openSUSE": "opensuse"
+  osinfo2common: |
+    "<rhv-operating-system>": "<vm-template>" <2>
+EOF
+----
+<1> Optional: You can change the value of the `namespace` parameter.
+<2> Specify the REST API name of the RHV operating system and its corresponding VM template according to the following examples:
+
+* `"rhel_6_9_plus_ppc64": "rhel6.9"`
+* `"rhel_6_ppc64": "rhel6.9"`
+* `"rhel_6": "rhel6.9"`
+* `"rhel_6x64": "rhel6.9"`
+* `"rhel_7_ppc64": "rhel7.7"`
+* `"rhel_7_s390x": "rhel7.7"`
+* `"rhel_7x64": "rhel7.7"`
+* `"rhel_8x64": "rhel8.1"`
+* `"sles_11_ppc64": "opensuse15.0"`
+* `"sles_11": "opensuse15.0"`
+* `"sles_12_s390x": "opensuse15.0"`
+* `"ubuntu_12_04": "ubuntu18.04"`
+* `"ubuntu_12_10": "ubuntu18.04"`
+* `"ubuntu_13_04": "ubuntu18.04"`
+* `"ubuntu_13_10": "ubuntu18.04"`
+* `"ubuntu_14_04_ppc64": "ubuntu18.04"`
+* `"ubuntu_14_04": "ubuntu18.04"`
+* `"ubuntu_16_04_s390x": "ubuntu18.04"`
+ifeval::["{HCOVersion}" != "2.4.0"]
+* `"windows_10": "win10"`
+* `"windows_10x64": "win10"`
+* `"windows_2003": "win10"`
+* `"windows_2003x64": "win10"`
+* `"windows_2008R2x64": "win2k8"`
+* `"windows_2008": "win2k8"`
+* `"windows_2008x64": "win2k8"`
+* `"windows_2012R2x64": "win2k12r2"`
+* `"windows_2012x64": "win2k12r2"`
+* `"windows_2016x64": "win2k16"`
+* `"windows_2019x64": "win2k19"`
+* `"windows_7": "win10"`
+* `"windows_7x64": "win10"`
+* `"windows_8": "win10"`
+* `"windows_8x64": "win10"`
+* `"windows_xp": "win10"`
+endif::[]
++
+The following ConfigMap example shows other Linux systems mapped to a `fedora31` template:
++
+[source,yaml]
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: os-configmap
+  namespace: default
+data:
+  osinfo2common: |
+    "other_linux": "fedora31"
+EOF
+----
+
+. Verify that the ConfigMap was created:
++
+----
+$ oc get cm -n default os-configmap -o yaml
+----
+
+ifeval::["{HCOVersion}" == "2.4.0"]
+. If you are importing a Windows VM:
 
-. If the RHV VM uses a Windows operating system, create a Windows {VirtProductName} template by running the following command:
+.. Create an {VirtProductName} Windows template:
```

Comment:
Just to verify, this command does not require any editing?

Reply:
Correct